index.html

```diff
@@ -47,132 +45,38 @@ <h1>Johnson Macharia</h1>
-          </ul>
-          <a href="#" class="see-project">See this project</a>
-        </div>
-      </div>
-
-      <div class="project-card project-card-2">
-        <img class="preview" src="icons/Rectangle 21 (1).png">
-        <div class="project-infor">
-          <p class="project-name">
-            Project name goes here.
-          </p>
-          <ul class="tech-list">
-            <li>html/css</li>
-            <li>Ruby on Rails</li>
-            <li>JavaScript</li>
-          </ul>
-          <a href="#" class="see-project">See this project</a>
-        </div>
-      </div>
 
-      <div class="project-card project-card-3">
-        <img class="preview" src="icons/Rectangle 21 (2).png">
-        <div class="project-infor">
-          <p class="project-name">
-            Project name goes here.
-          </p>
-          <ul class="tech-list">
-            <li>html/css</li>
-            <li>Ruby on Rails</li>
-            <li>JavaScript</li>
-          </ul>
-          <a href="#" class="see-project">See this project</a>
-        </div>
-      </div>
-
-      <div class="project-card project-card-4">
-        <img class="preview" src="icons/Rectangle 21 (3).png">
-        <div class="project-infor">
-          <p class="project-name">
-            Project name goes here.
-          </p>
-          <ul class="tech-list">
-            <li>html/css</li>
-            <li>Ruby on Rails</li>
-            <li>JavaScript</li>
-          </ul>
-          <a href="#" class="see-project">See this project</a>
-        </div>
-      </div>
-
-      <div class="project-card project-card-5">
-        <img class="preview" src="icons/Rectangle 21 (4).png">
-        <div class="project-infor">
-          <p class="project-name">
-            Project name goes here.
-          </p>
-          <ul class="tech-list">
-            <li>html/css</li>
-            <li>Ruby on Rails</li>
-            <li>JavaScript</li>
-          </ul>
-          <a href="#" class="see-project">See this project</a>
-        </div>
-      </div>
-
-      <div class="project-card project-card-6">
-        <img class="preview" src="icons/Rectangle 21 (5).png">
-        <div class="project-infor">
-          <p class="project-name">
-            Project name goes here.
-          </p>
-          <ul class="tech-list">
-            <li>html/css</li>
-            <li>Ruby on Rails</li>
-            <li>JavaScript</li>
-          </ul>
-          <a href="#" class="see-project">See this project</a>
-        </div>
-      </div>
+    <section id="project-container" class="">
+      <h2>Projects</h2>
+      <div class="dec-2"><img class="decoration" src="icons/image-geometry_2 1.png"></div>
+      <div class="dec-3"><img class="decoration" src="icons/image-geometry_3.png"></div>
+      <div class="dec-4"><img class="decoration" src="icons/image-geometry_4.png"></div>
+      <div class="dec-5"><img class="decoration" src="icons/image-geometry_5.png"></div>
```

Comment:
- [ ] In your desktop version, add the icons missing on the buttons
- [ ] Add the arrows missing in your `Previous project` and `Next project` buttons

![image](https://github.com/JohnteFresh/Portfolio-Development/assets/93630700/1df86305-80d2-4b6c-b031-d7087487fbae)

![image](https://github.com/JohnteFresh/Portfolio-Development/assets/93630700/f933bd59-1148-405e-b16b-d25f8326fda1)